code/game/objects/items/weapons/autolathe_disks.dm

```diff
@@ -463,8 +463,9 @@
 /obj/item/weapon/computer_hardware/hard_drive/portable/design/onestar
 	disk_name = "One Star Tool Pack"
 	icon_state = "onestar"
-	rarity_value = 50
+	rarity_value = 70
```

Comment:
Should it be spawned in regular maints and how would it affect lockers at one star dungeon?

Reply:
this will now be given less priority than other one star technology pieces. how are one star stock parts or prothesis.

Reply:
I would ask to make its value to 100 or just blacklist it whole

In my experience those parts are found from consoles but let me check it on my local once more